shape_optimization/10_airfoil_optimization_with_SU2/run_optimization.py

```diff
@@ -0,0 +1,63 @@
+# Import Kratos core and apps
+import KratosMultiphysics as KM
+import KratosMultiphysics.ShapeOptimizationApplication as KSO
+
+# Additional imports
+from KratosMultiphysics.ShapeOptimizationApplication import optimizer_factory
+from KratosMultiphysics.ShapeOptimizationApplication.interface_su2 import InterfaceSU2
+from KratosMultiphysics.ShapeOptimizationApplication.analyzer_base import AnalyzerBaseClass
+
+# =======================================================================================================
+# Define external analyzer
+# =======================================================================================================
+
+with open("parameters.json",'r') as parameter_file:
+    parameters = KM.Parameters(parameter_file.read())
+
+interface_su2 = InterfaceSU2(parameters["su2_interface_settings"])
+#interface_su2.WriteSU2MeshAsMDPA()
```

Comment:
for the example in the repo i would keep this command activated and not commit the mdpa file. 
Adding a comment to this line, that technically the translation needs to be done only once can help for understanding.